.travis.yml

```diff
@@ -5,6 +5,11 @@ node_js:
   - '8'
 install:
   - travis_wait 60 npm install
+  - travis_wait 60 npm run get-client
+  - travis_wait 60 npm run get-core-apps
+  - travis_wait 60 npm run get-dao-templates
+  - travis_wait 60 npm run get-fundraising
+  - travis_wait 60 npm run get-open-enterprise
```

Comment:
Note that there is going to be a limit on the max build time for Travis regardless of this; the maximum for open source projects is 50 min (up to 120min on the paid plans) :(.

Reply:
Oh that's not cool. Nevertheless I been think that having a solution as discussed [here](https://github.com/aragon/aragon-cli/issues/627) makes more and more sense as more devs start building their own apps.

Maybe we can have a command to generate a snapshot from an aragonPM instance 🤔

Reply:
Yes! That would be super cool; and then we can just publish to aragen using the aragonPM snapshot's latest version content :).